pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -92,6 +92,11 @@ sleep 5
 done`
 )
 
+const (
+	redColor   = "\u001b[31;1m"
```

Comment:
This is unix specific and I'd not prefer to add something os specific.

Reply:
Hi, thanks for having a look at this. I'll see if I can find something platform agnostic without pulling in too many dependencies. However, just to make you aware of this:

![image](https://github.com/openshift/oc/assets/3291433/6dd6e10e-b3d5-4265-bd6a-ea3586fefcde)

That color is coming from vendor/k8s.io/client-go/rest/warnings.go which is doing the exact same thing:
```
  102 const (                                                                                                                 
  103     yellowColor = "\u001b[33;1m"                                                                                        
  104     resetColor  = "\u001b[0m"                                                                                           
  105 )                                                                                                                       
  106                                                                                                                         
  107 // HandleWarningHeader prints warnings with code=299 to the configured writer.                                          
  108 func (w *warningWriter) HandleWarningHeader(code int, agent string, message string) {                                   
  109     if code != 299 || len(message) == 0 {                                                                               
  110         return                                                                                                          
  111     }                                                                                                                   
  112                                                                                                                         
  113     w.writtenLock.Lock()                                                                                                
  114     defer w.writtenLock.Unlock()                                                                                        
  115                                                                                                                         
  116     if w.opts.Deduplicate {                                                                                             
  117         if _, alreadyWritten := w.written[message]; alreadyWritten {                                                    
  118             return                                                                                                      
  119         }                                                                                                               
  120         w.written[message] = struct{}{}                                                                                 
  121     }                                                                                                                   
  122     w.writtenCount++                                                                                                    
  123                                                                                                                         
  124     if w.opts.Color {                                                                                                   
  125         fmt.Fprintf(w.out, "%sWarning:%s %s\n", yellowColor, resetColor, message)                                       
  126     } else {                                                                                                            
  127         fmt.Fprintf(w.out, "Warning: %s\n", message)                                                                    
  128     }                                                                                                                   
  129 }  
```